File: BubbleSort-algorithm.js


var unsorted = [5,2,6,3,1,3]
var unsorteddd = [5,3,8,1,7,10,11,1,9]




function sortList(list){
	for (i=0; i < unsorted.length; i++){
		var fullSort = false;
		var loopCount = 1;
		for (j=0; j < unsorted.length-1; j++){
			if(list[j] > list[j+1]){
				fullSort = true;
				var x = list[j+1]
				list[j+1] = list[j];
				list[j] = x;
				
			}
		loopCount++
		console.log(loopCount)
		if (fullSort == false);
		

		}
	}
	
	console.log(list);
	// return list;
}

sortList(unsorteddd);



// QUICK SORT
function swap(items, firstIndex, secondIndex){
    var x = items[firstIndex];
    items[firstIndex] = items[secondIndex];
    items[secondIndex] = temp;
}

File: BubbleSort-algorithm.py
unsorted = [5,2,6,3,1,3];

def sort_list(the_list):
	for i in xrange(0,len(unsorted)): 
		full_sort = False;
		loop_count = 1;
		for j in xrange(0,len(unsorted) - 1):
			if (the_list[j] > the_list[j+1]):
				temp = the_list[j];
				the_list[j] = the_list[j+1];
				the_list[j+1] = temp;

		loop_count += 1
		full_sort = False;


	return the_list;

print sort_list(unsorted)



	